<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntityWidget</class>
 <widget class="QWidget" name="EntityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblEntityStates">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Entity States:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbStates">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="btnAddState">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveState">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnRenameState">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnOrderStateBack">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnOrderStateForward">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkStartEnabled">
     <property name="text">
      <string>Start Enabled</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="ChildrenToolbar">
       <item>
        <widget class="QToolButton" name="btnAddChild">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnAddBoundingVolume">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnAddPhysics">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnAddScissor">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QTreeView" name="childrenTree"/>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="pageNull">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="lblNoChildSelected">
            <property name="text">
             <string>No child selected</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageSprite">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageFont"/>
        <widget class="QWidget" name="pagePhysics">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QGroupBox" name="grpPhysics">
            <property name="title">
             <string>Enable Physics</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Type:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Static</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Kinematic</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dynamic</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Gravity Scale:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox">
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Linear Damping:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QSlider" name="horizontalSlider_2">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Angular Damping:</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QSlider" name="horizontalSlider">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QCheckBox" name="chkDynamicCcd">
               <property name="text">
                <string>Dynamic CCD</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QCheckBox" name="chkFixedRot">
               <property name="text">
                <string>Fixed Rotation</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QCheckBox" name="chkInitiallyAwake">
               <property name="text">
                <string>Initially Awake</string>
               </property>
              </widget>
             </item>
             <item row="10" column="1">
              <widget class="QCheckBox" name="chkAllowSleep">
               <property name="text">
                <string>Allow Sleep</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageBoundingVolume">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Bounding Volumes</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QComboBox" name="comboBox_2"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRenameState">
   <property name="icon">
    <iconset resource="../res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-rename.png</normaloff>:/icons16x16/generic-rename.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rename State</string>
   </property>
   <property name="toolTip">
    <string>Rename state</string>
   </property>
  </action>
  <action name="actionAddState">
   <property name="icon">
    <iconset resource="../res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add State</string>
   </property>
   <property name="toolTip">
    <string>Add a new state for this entity</string>
   </property>
  </action>
  <action name="actionRemoveState">
   <property name="icon">
    <iconset resource="../res/DesignerTool.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remo&amp;ve State</string>
   </property>
   <property name="toolTip">
    <string>Remove the currently selected state</string>
   </property>
  </action>
  <action name="actionOrderStateBackwards">
   <property name="icon">
    <iconset resource="../res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-left.png</normaloff>:/icons16x16/generic-left.png</iconset>
   </property>
   <property name="text">
    <string>Order State &amp;Backwards</string>
   </property>
   <property name="toolTip">
    <string>Shift the index of the current state back by 1</string>
   </property>
  </action>
  <action name="actionOrderStateForwards">
   <property name="icon">
    <iconset resource="../res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-right.png</normaloff>:/icons16x16/generic-right.png</iconset>
   </property>
   <property name="text">
    <string>Order State For&amp;wards</string>
   </property>
   <property name="toolTip">
    <string>Shift the index of the current state forward by 1</string>
   </property>
  </action>
  <action name="actionInsertPrimitive">
   <property name="icon">
    <iconset resource="../res/DesignerTool.qrc">
     <normaloff>:/icons16x16/media-pauseRecord.png</normaloff>:/icons16x16/media-pauseRecord.png</iconset>
   </property>
   <property name="text">
    <string>Insert Primitive</string>
   </property>
   <property name="toolTip">
    <string>Inserts a child primitive at the selected node</string>
   </property>
  </action>
  <action name="actionInsertBoundingVolume">
   <property name="text">
    <string>Insert Bounding Volume</string>
   </property>
   <property name="toolTip">
    <string>Inserts a bounding volume at the selected node to detect collision, mouse input, hitbox, etc.</string>
   </property>
  </action>
  <action name="actionInsertPhysicsBody">
   <property name="text">
    <string>Insert Physics Body</string>
   </property>
   <property name="toolTip">
    <string>Inserts a physics body at the selected node that simulates forces and velocity</string>
   </property>
  </action>
  <action name="actionTemp">
   <property name="text">
    <string>Temp</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/DesignerTool.qrc"/>
 </resources>
 <connections/>
</ui>
